resolwe_bio/docker_images/dnaseq/README.md

```diff
@@ -14,5 +14,6 @@ Included bioinformatics tools:
 * Picard Tools (v2.8.1+dfsg-3)
 * Primerclip (v180308)
 * Rsamtools (v1.30.0)
+* SnpEFF (4.3T)
```

Comment:
The official website and GitHub use `SnpEff`, SourceForge uses `snpEff`. I would stick to one of those choices.